compiler/rustc_parse/src/parser/path.rs

```diff
@@ -272,7 +272,15 @@ impl<'a> Parser<'a> {
                         lo,
                         ty_generics,
                     )?;
-                    self.expect_gt()?;
+                    if self.token.is_ident()
+                        || self.token.is_lifetime()
+                        || self.token.is_numeric_lit()
+                    {
+                        // The current token is probably part of the generic arguments and simply wasn't properly separated by a comma.
```

Comment:
This makes it so that you suggest adding a comma in `<A, B C>` -> `<A, B, C>`?

Reply:
Yea, exactly